config/settings.yml

```diff
@@ -1439,10 +1439,21 @@ genisis:
   pass: bogus
 
 # Settings for connecting AFS Veteran Services
+# For locahost we can use the existing certs as long as we don't call out
 dgi:
+  # add med_api here? Will need to reach out to that team
   jwt:
-    public_key_path: modules/meb_api/spec/fixtures/dgi_public_test.pem
-    private_key_path: modules/meb_api/spec/fixtures/dgi_private_test.pem
+    public_key_path: ""
+    private_key_path: ""
+  vye:
+    jwt:
+      # May not need the public path
+      public_key_path: "/srv/vets-api/secret/dgib_jwt.key"
```

Comment:
Please use empty strings, not the real values

Reply:
Changed to empty strings